tests/cloner_test.go

```diff
@@ -381,8 +381,12 @@ var _ = Describe("all clone tests", func() {
 								sourcePV.SetLabels(make(map[string]string))
 							}
 							sourcePV.GetLabels()["source-pv"] = "yes"
-							sourcePV, err = f.K8sClient.CoreV1().PersistentVolumes().Update(context.TODO(), sourcePV, metav1.UpdateOptions{})
-							Expect(err).ToNot(HaveOccurred())
+							Eventually(func() error {
+								// We shouldn't make the test fail if there's a conflict with the update request.
+								// These errors are usually transient and should be fixed in subsequent retries.
+								sourcePV, err = f.K8sClient.CoreV1().PersistentVolumes().Update(context.TODO(), sourcePV, metav1.UpdateOptions{})
```

Comment:
Should we refresh the sourcePV via a GET call? I'm thinking that if this is a conflict, it'll never resolve unless we get the updated resource?

Reply:
Oops good catch

Reply:
Unless I am missing something, this iteration does not refresh the PV object from the API server